api/api.js

```diff
@@ -31,12 +32,15 @@ const CLUSTER_INFO_BROADCAST_INTERVAL_MS = 5000;
 const CLUSTER_INFO_CACHE_TIME_SECS = 4500;
 const CONFIG_PROGRAM_ID = 'Config1111111111111111111111111111111111111';
 const MAX_KEYBASE_USER_LOOKUP = 50;
+const GEO_IP_KEY = '2504e6097a8048c585560d1ebeff00c2';
```

Comment:
i will reset it, pls create your own and set it as required.

Reply:
Can we do something like:
https://github.com/solana-labs/tour-de-sol/blob/1b2e1f041a0a52d5f28d01c8071a9e3fb49a0480/ip-collector/audit.js#L7-L13
except that when there's no key just emit a warning and disable geolocation

Reply:
@mvines done